grpc_tensorflow_server/grpc_tensorflow_server.py

```diff
@@ -99,9 +99,16 @@ def main(_):
   if not FLAGS.job_name:
     raise ValueError("Empty job_name")
   server_def.job_name = FLAGS.job_name
-
+  job_def = None
+  for i in range(len(server_def.cluster.job)):
```

Comment:
What is the motivation for this change? Did you actually encounter a case where the job name wasn't part of the cluster spec? The cluster spec should be autogenerated so I think that would indicate a bug someplace else as well?

Reply:
I just want to make this code more robust.